scripts/cmscp.py

```diff
@@ -485,6 +485,10 @@ def perform_local_stageout(local_stageout_mgr, \
         ## Fallback to previous behaviour where phedex is queried for location
         if source_site == 'unknown':
             source_site = G_NODE_MAP.get(dest_temp_se, 'unknown')
+        
+        # If fallback stageout happens, PNN can be different as source
+        if 'PNN' in stageout_info and stageout_info['PNN'] != source_site:
```

Comment:
I think this part of the if is redundant: `and stageout_info['PNN'] != source_site`

Let's always set the source site to what we got from the stageout manager